Add changelog entry for consensus cell types (#406)
AlexsLemonade/scpca-docs: docs/CHANGELOG.md

<!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
<!-------------------------------------------------->

## UPDATE WITH DATE FOR RELEASE 

* Consensus cell type annotations are now available in all processed `SingleCellExperiment` and `AnnData` objects and merged objects. 

>   * Annotations from `SingleR` and `CellAssign` are used to assign an ontology-aware consensus cell type label. 

Reviewer: I tried to rephrase this to make a bit more explicitly informational, what do you think?

```suggestion
  * These consensus annotations are ontology-aware consensus cell type labels derived from `SingleR` and `CellAssign` labels. 
```

Author: ```suggestion
  * The labels obtained from `SingleR` and `CellAssign` are used to assign an ontology-aware consensus cell type label. 
```
I tried a slightly different rephrasing that I think I like a little better.

>   * See our {ref}`documentation on cell type annotation<processing_information:cell type annotation>` to learn more about how consensus cell types are assigned. 
>   * For more information on locating these cell type annotations in the downloaded objects see {ref}`the single-cell gene expression file contents page<sce_file_contents:singlecellexperiment cell metrics>` and {ref}`the merged object file contents page<merged_objects:singlecellexperiment cell metrics>`.

Reviewer: Just making this parallel

```suggestion
  * See our {ref}`documentation on cell type annotation<processing_information:cell type annotation>` to learn more about how consensus cell types are assigned. 
  * See {ref}`the single-cell gene expression file contents page<sce_file_contents:singlecellexperiment cell metrics>` and {ref}`the merged object file contents page<merged_objects:singlecellexperiment cell metrics>` for more information on obtaining these cell type annotations from the downloaded objects .
```

>   * See {ref}`the single-cell gene expression file contents page<sce_file_contents:singlecellexperiment cell metrics>` and {ref}`the merged object file contents page<merged_objects:singlecellexperiment cell metrics>` for more information on obtaining these cell type annotations from the downloaded objects .

Reviewer: ```suggestion
  * See {ref}`the single-cell gene expression file contents page<sce_file_contents:singlecellexperiment cell metrics>` and {ref}`the merged object file contents page<merged_objects:singlecellexperiment cell metrics>` for more information on obtaining these cell type annotations from the downloaded objects.
```
